### PR TITLE
Docs Update

### PR DESCRIPTION
**Pull request by humanagent**
Updated docs 
- Quickstart
- Frames.js
- Frog
- OnchainKit

### CODE REVIEW

File: pages/quickstart/frames-js.mdx [hunk 1]
```diff
@@ -3,10 +3,52 @@ title: "Open Frames - frames.js"
 description: "An extension of Open Graph that makes link previews interactive."
 ---
 
-# frames.js
+# Frames.js
 
-[frames.js](https://framesjs.org) is a TypeScript framework for building frames with support for Open Frames.
+Discover how Frog seamlessly incorporates XMTP payloads
```
Comment (davidfurlong) [inline pages/quickstart/frames-js.mdx]: Frames.js

File: pages/quickstart/frames-js.mdx [hunk 2]
```diff
@@ -3,10 +3,52 @@ title: "Open Frames - frames.js"
...
 
-## Learn more
+- [Official Docs](https://framesjs.org/reference/js/xmtp): Official Frames.js Documentation.
+- [Quickstart](https://github.com/ephemeraHQ/open-frames-starter-framesjs/): OnchainKit quickstart that integrates XMTP.
```
Comment (davidfurlong) [inline pages/quickstart/frames-js.mdx]: Frames.js quickstart

File: pages/quickstart/frames-js.mdx [hunk 3]
```diff
@@ -3,10 +3,52 @@ title: "Open Frames - frames.js"
...
 ---
 
-# frames.js
+# Frames.js
 
-[frames.js](https://framesjs.org) is a TypeScript framework for building frames with support for Open Frames.
+Discover how Frog seamlessly incorporates XMTP payloads
 
-## Learn more
+- [Official Docs](https://framesjs.org/reference/js/xmtp): Official Frames.js Documentation.
```
Comment (davidfurlong) [inline pages/quickstart/frames-js.mdx]: link is outdated; https://framesjs.org/guides/open-frames is probably best atm

File: pages/quickstart/frames-js.mdx [hunk 4]
```diff
@@ -3,10 +3,52 @@ title: "Open Frames - frames.js"
...
+Now validate the incoming message.
+
+```jsx
+let fid: number | undefined;
+let walletAddress: string | undefined;
+
+import {
+  isXmtpFrameRequest,
+  getXmtpFrameMessage,
+} from "@coinbase/onchainkit/xmtp";
```
Comment (davidfurlong) [inline pages/quickstart/frames-js.mdx]: wrong lib & example

**Review by davidfurlong — CHANGES_REQUESTED** [review]

**Comment by humanagent** [comment]
Thks @davidfurlong ! Changes made

File: pages/quickstart/frames-js.mdx [hunk 5]
```diff
@@ -3,10 +3,77 @@ title: "Open Frames - frames.js"
...
 
-# frames.js
+# Frames.js
 
-[frames.js](https://framesjs.org) is a TypeScript framework for building frames with support for Open Frames.
+Discover how Frames.js seamlessly incorporates XMTP payloads
 
-## Learn more
+- [Official Docs](https://framesjs.org/guides/open-frames): Official Frames.js Documentation.
+- [Frames.js quickstart](https://github.com/ephemeraHQ/open-frames-starter-framesjs/): OnchainKit quickstart that integrates XMTP.
```
Comment (davidfurlong) [inline pages/quickstart/frames-js.mdx]: `OnchainKit quickstart` -> `Frames.js quickstart`

**Review by davidfurlong — APPROVED** [review]
@fabriguespe I made some changes to the frames.js guide directly

File: pages/quickstart/frames-js.mdx (hunk 5 above)
Comment (davidfurlong) [reply pages/quickstart/frames-js.mdx]: Outdated

**Comment by humanagent** [comment]
@davidfurlong nice. I dont have access to the repo or team to be able to merge.

**Pull request merged**